.golangci.yml

```diff
@@ -87,34 +87,9 @@ linters-settings:
     # To select a subset of rules to run.
     # Available rules: https://github.com/securego/gosec#available-rules
     # Default: [] - means include all rules
-    includes:
-      - G101
-      - G102
-      - G103
-      - G106
-      - G107
-      - G108
-      - G109
-      - G110
-      - G111
-      - G112
-      - G114
-      - G201
-      - G202
-      - G203
-      - G301
-      - G302
-      - G303
-      - G305
-      - G306
-      - G401
-      - G403
-      - G404
-      - G501
-      - G502
-      - G503
-      - G505
-      - G601
+    includes: [ G101, G102, G103, G106, G107, G108, G109, G110, G111, G112, G114, G201, G202, G203, G301, G302, G303, G305, G306,
+                G401, G403, G404, G501, G502, G503, G505, G601 ]
+    # G104, G105, G113, G204, G304, G307, G402, G504 were not enabled intentionally
```

Comment:
Can we use this PR to only add the comment? I think I prefer the longer list to track down the PRs that added them

Reply:
I can, but I think that we already have place to track these PRs: https://github.com/influxdata/telegraf/issues/12957

Reply:
That is true and nice to have, however, it is also nice to be able to view the git history via blame and see what commit added what without needing to open up github :)

One of my complaints about linter changes is it makes doing a git blame a lot harder than it needs to be sometimes.

Reply:
I also like the list with 1 item per line. 

But indeed we have a tracking in #12957 for current list. It might be less clear if some changes to the list are done in the future.

Reply:
@powersj Makes sense!